circle.yml

```diff
@@ -57,6 +57,11 @@ dependencies:
     - php -f scripts/slack/slack_notify.php pantheon_create "${TERMINUS_ENV}" "${BRANCH}"
     - terminus build-env:delete:ci -n "$TERMINUS_SITE" --keep=2 --yes
     - composer -n build-assets
+    - |
+      cd web/themes/custom/badcamp2017/
+      npm install -g gulp
+      npm install
+      gulp sass
```

Comment:
Not to bikeshed (this certainly works), but we could add also add this to the composer build-assets command.  We'd drop it into a shell script in scripts, then add it to the array in composer.json -> scripts -> build-assets.